valhalla/baldr/graphconstants.h

```diff
@@ -359,12 +359,15 @@ inline std::string to_string(Use u) {
 
 enum class TaggedValue : uint8_t { // must start at 1 due to nulls
   kLayer = 1,
+  kPronunciation = 2,
   // we used to have bug when we encoded 1 and 2 as their ASCII codes, but not actual 1 and 2 values
   // see https://github.com/valhalla/valhalla/issues/3262
   kTunnel = static_cast<uint8_t>('1'),
   kBridge = static_cast<uint8_t>('2'),
 };
 
+enum class PronunciationAlphabet : uint8_t { kIpa = 0, kXKatakana = 1, kXJeita = 2, kNtSampa = 3 };
```

Comment:
sorry put this comment on the wrong pr, getting confused whats what! id move the different phonetic alphabets to be the tag in the taggedvalue class. sure it uses 4 instead of one but it really reduces the complexity and size required to store the info